### PR TITLE
topics: add open-policy-agent

### PR DESCRIPTION
**Pull request by gshively11**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

For the sake of transparency, I am an active contributor to the OPA project, but not a maintainer/creator/contractor/employee. I wasn't sure if that constituted an "affiliation" per the verbiage above.

### Which change are you proposing?

  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Open Policy Agent is a [widely adopted](https://github.com/open-policy-agent/opa), Cloud Native Computing Foundation [graduated project](https://www.cncf.io/announcements/2021/02/04/cloud-native-computing-foundation-announces-open-policy-agent-graduation/).

### CODE REVIEW

**Comment by gshively11** [comment]
My local ruby environment is wonky right now, so I'm relying on the PR for CI.  Once someone approves the workflow run I'll check the results and make any necessary tweaks.

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Review by kenyonj — CHANGES_REQUESTED** [review]
Topic name, directory, and logo filename all have to be the same, and then the aliases cannot include the topic name.

File: topics/open-policy-agent/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The Open Policy Agent (OPA, pronounced “oh-pa”) is an open source, general-purpose policy engine that unifies policy enforcement across the stack. OPA provides a high-level declarative language that lets you specify policy as code and simple APIs to offload policy decision-making from your software. You can use OPA to enforce policies in microservices, Kubernetes, CI/CD pipelines, API gateways, and more.
+
+### Community & Support
+- [Website](https://www.openpolicyagent.org/)
+- [GitHub](https://github.com/open-policy-agent/)
+- [Documentation](https://www.openpolicyagent.org/docs/latest/)
+
+**What is OPA**
+
+[//youtube-embed-unfurl//]: # (ijQdHy4XQCU)
```
Comment (nholden) [inline topics/open-policy-agent/index.md:20]: I'm not seeing anything in the rendered Markdown for this section. What are you intending to include?

Comment (gshively11) [reply topics/open-policy-agent/index.md:20]: I was trying to replicate the pattern on the [actions topic](https://raw.githubusercontent.com/github/explore/main/topics/actions/index.md) (see the bottom line). But it doesn't seem to be rendering there either.  It's meant to be an introductory video to OPA: https://www.youtube.com/watch?v=ijQdHy4XQCU&t.  I'm fine with removing it entirely, or if there's a better way to render video, we can do that, or maybe just link to it?  Just let me know.

Comment (nholden) [reply topics/open-policy-agent/index.md:20]: Oh, interesting! Let's give it a shot!

**Review by nholden — APPROVED** [review]

**Review by nholden — CHANGES_REQUESTED** [review]
Unfortunately, it looks like you still have some test failures. Could you address them?

```
  1) Failure:
topics::open-policy-agent topic#test_0019_has at most one image with the right name, type, and dimensions [/home/runner/work/explore/explore/test/topics_test.rb:229]:
topic images should be 288px wide.
Expected: 288
  Actual: 1801

  2) Failure:
topics::open-policy-agent topic#test_0017_has matching logo, when logo key exists [/home/runner/work/explore/explore/test/topics_test.rb:198]:
should not specify logo opa.png when file does not exist

  3) Failure:
topics::open-policy-agent topic#test_0018_has a matching logo key, when logo exists [/home/runner/work/explore/explore/test/topics_test.rb:212]:
logo key should be open-policy-agent.png, but was opa.png
```

**Review by nholden — APPROVED** [review]

**Comment by nholden** [comment]
Thanks very much for your contribution, @gshively11! The video [looks great](https://admin.github.com/topics/open-policy-agent). 💖

**Pull request merged**